spec/safe_statements_spec.rb

```diff
@@ -1279,6 +1279,24 @@ def up
           end
         end
 
+        describe "unsafe_remove_check_constraint" do
+          ["constraint_foo_bar_is_not_null", :constraint_foo_bar_is_not_null].each do |constraint_name|
+            it "does not throw an error when #{constraint_name.is_a?(Symbol) ? "symbol" : "string"} is used for contraint name" do
+              migration = Class.new(migration_klass) do
+                define_method(:up) do
+                  unsafe_create_table :foos
+                  unsafe_add_column :foos, :bar, :text
+                  unsafe_add_check_constraint :foos, "bar IS NOT NULL", :name => :constraint_foo_bar_is_not_null
+                  unsafe_remove_check_constraint :foos, name: constraint_name
+                end
+              end
+              expect do
+                migration.suppress_messages { migration.migrate(:up) }
+              end.not_to raise_error
```

Comment:
Can we add an assertion that the constraint is removed? That might mean making the `unsafe_add_check_constraint` (and prior) DDL part of setup and the assertions about only the remove call.

Reply:
Updated: https://github.com/braintree/pg_ha_migrations/pull/102/commits/c52eaecdaaf79e21119b3d141f6347dc1b1d67aa